tests/routes/test_vespasearch.py

```diff
@@ -259,39 +257,33 @@ def test_specific_doc_returned(test_vespa, monkeypatch, data_client, data_db):
 
 
 @pytest.mark.search
-@pytest.mark.parametrize(
-    "params",
-    [
-        SearchParameters(query_string="climate"),
-        SearchParameters(query_string="climate", exact_match=True),
-        SearchParameters(
-            query_string="climate",
-            exact_match=True,
-            limit=1,
-            max_hits_per_family=10,
-        ),
-    ],
-)
-def test_search_params_contract(
-    params, test_vespa, monkeypatch, data_client, data_db, mocker
+def test_search_params_backend_limits(
+    test_vespa, monkeypatch, data_client, data_db, mocker
 ):
+    search_limit = 10
+    passage_limit = 5
+
     monkeypatch.setattr(search, "_VESPA_CONNECTION", test_vespa)
+    monkeypatch.setattr(search_schemas, "VESPA_SEARCH_LIMIT", search_limit)
+    monkeypatch.setattr(search_schemas, "VESPA_SEARCH_MATCHES_PER_DOC", passage_limit)
     _populate_db_families(data_db)
     query_spy = mocker.spy(search._VESPA_CONNECTION, "search")
 
     _make_search_request(
         data_client,
         params={
-            "query_string": params.query_string,
-            "exact_match": params.exact_match,
-            "limit": params.limit,
-            "max_hits_per_family": params.max_hits_per_family,
+            "query_string": "the",
```

Comment:
Have we lost the ability to do an "exact match" search?

Reply:
We only lost it from this test. I realised the limits where the only things that still needed coverage from what this test was testing previously after consolidating the model, so I converted this test. The exact match parameter wasn't really adding anything to the test coverage here so I just removed it for simplicity. (It still works as a query type though and we have test coverage for exact matches elsewhere)